oe_theme.theme

```diff
@@ -632,8 +662,24 @@ function oe_theme_preprocess_links__language_block(array &$variables): void {
       $language['active'] = TRUE;
     }
 
-    $variables['languages'][] = $language;
+    // Take language categories and order languages accordingly.
+    $language_config = $language_config_storage->load($language_code);
+    $category = $language_config->getThirdPartySetting('oe_multilingual', 'category', 'no_category');
+    switch ($category) {
+      case 'eu':
+        $variables['eu_languages'][] = $language;
+        break;
+
+      case 'non_eu':
+        $variables['non_eu_languages'][] = $language;
+        break;
+
+      case 'no_category':
```

Comment:
Mmm, let's not use our own thing here and use a `default` case. Meaning: if it's not eu or non_eu, it goes in this bucket. So that if the category is set as banana, it's not a valid category. No?

Reply:
Yes, I was hesitating with this and default, because if it is anything else it won't go anywhere not even this bucket as you say, for banana. For me it's okay like that as well.